NetCasbin/InternalEnforcer.cs

```diff
@@ -17,34 +17,26 @@ public class InternalEnforcer : CoreEnforcer
         /// <returns></returns>
         protected Boolean AddPolicy(String sec, String ptype, List<String> rule)
         {
-            Boolean ruleAdded = model.AddPolicy(sec, ptype, rule);
-            if (!ruleAdded)
-            {
-                return false;
-            } 
-
             if (adapter != null && autoSave)
```

Comment:
I suggest call model.HasPolicy to check if the policy can be added to adapter.

Reply:
You are right, I changed it.